src/editor/Core/Utility.re

```diff
@@ -43,3 +43,13 @@ let safe_fold_left2 = (fn, accum, list1, list2, ~default) =>
     print_endline("fold_left2 failing because: " ++ reason);
     default;
   };
+
+let join = paths => {
```

Comment:
Cool! I like this idea for `Rench`, too (I just made a `Path.join` in that project, but it only took two arguments... I like the list idea).